OpenSim/Common/XMLDocument.cpp

```diff
@@ -64,7 +64,8 @@ using namespace std;
 // 30514 for removing "reverse" property from Joint
 // 30515 for WrapObject color, display_preference, VisibleObject -> Appearance
 // 30516 for GeometryPath default_color -> Appearance
-const int XMLDocument::LatestVersion = 30516;
+// 30517 for removal of Socket _connectee_name from XML markup for Socket
```

Comment:
It sounds a bit like the Socket was removed from XML, but the XML tag name was just shortened.